### PR TITLE
Support `poolName` or `poolID` validation in Azure Pipelines Scaler

### PR DESCRIPTION
**Pull request by JorTurFer**
Signed-off-by: jorturfer <jorge_turrado@hotmail.es>

_Provide a description of what has been changed_

### Checklist

- [x] Commits are signed with Developer Certificate of Origin (DCO - [learn more](https://github.com/kedacore/keda/blob/main/CONTRIBUTING.md#developer-certificate-of-origin-signing-your-work))
- [x] Tests have been added
- [x] A PR is opened to update our Helm chart ([repo](https://github.com/kedacore/charts)) *(if applicable, ie. when deployment manifests are modified)*
- [x] A PR is opened to update the documentation on ([repo](https://github.com/kedacore/keda-docs)) *(if applicable)* https://github.com/kedacore/keda-docs/pull/612
- [x] Changelog has been updated

Relates to #2342

### CODE REVIEW

**Comment by tomkerkhove** [comment]
Would you mind adding an issue for this @JorTurFer? Instead of using one or the other, maybe we should allow them to configure both and use what they want to use?

**Comment by tomkerkhove** [comment]
Ok this is probably related to https://github.com/kedacore/keda/issues/2342

**Comment by JorTurFer** [comment]
> Ok this is probably related to https://github.com/kedacore/keda/issues/2342

Yes it is, I forgot adding the link :(

File: CHANGELOG.md [hunk 1]
```diff
@@ -23,7 +23,7 @@
 
 ### New
 
-- TODO ([#XXX](https://github.com/kedacore/keda/pull/XXX))
+- Support `poolName` or `poolID` validation in Azure Pipelines Scaler ([#2370](https://github.com/kedacore/keda/pull/2370))
```
Comment (zroubalik) [inline CHANGELOG.md:26]:
```suggestion
- Azure Pipelines Scaler: support `poolName` or `poolID` validation ([#2370](https://github.com/kedacore/keda/pull/2370))
```

And move this to Improvements section please

**Review by zroubalik — COMMENTED** [review]
Nit in Changelog, the rest looks good.

**Pull request merged**